cmake/tpl/conduit.cmake

```diff
@@ -6,9 +6,9 @@ set(CONDUIT_CACHE "${CACHE_DIR}/${CONDUIT_DIST}")
 list(APPEND ${lib_name}_INCLUDES $<BUILD_INTERFACE:${${lib_name}_DIR}/include/${lib_name}>)
 
 set(${lib_name}_libs 
-    libconduit.so
-    libconduit_blueprint.so
-    libconduit_relay.so
+    libconduit.a
+    libconduit_blueprint.a
+    libconduit_relay.a
```

Comment:
I like this fix over changing the library rpaths. We will need to rebuild the LC TPL directories first before merging this however.

Reply:
This is also how Blast installs the libraries and is consistent with what axom does in Spheral. I'm not sure why the defaults are different between the two packages.